src/components/tab-switcher/tab-switcher.jsx

```diff
@@ -36,20 +36,27 @@ export default React.createClass({
   },
   tabClick: function (index) {
     this.setState({activeTab: index});
+
+    if (this.props.onChange) {
+      this.props.onChange({
+        index: index,
+        tabName: this.props.children[index].props.name
+      });
+    }
   },
   render: function() {
     let buttons = this.props.children.map((element, index) => {
       if(this.props.children[index].props.hidden) { return; }
+
       return (
-        <Link
-          to={`${this.props.baseURL}${this.props.children[index].props.slug}`}
+        <button
           className={`btn ${index === this.state.activeTab ? `active` : ``}`}
           onClick={this.tabClick.bind(null, index)}
           key={index}
```

Comment:
based on the code later in the PR, maybe `element.getAttibute('slug')`? I'm not actually sure what the best key is here.